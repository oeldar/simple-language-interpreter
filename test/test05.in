a := 10;
print(a);
b := a + 10;
print(b);
print(c);
a := b := c := 100 * 5 + 5;
print(a);
print(b);
print(c);
