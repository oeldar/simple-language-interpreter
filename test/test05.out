10
20
0
505
505
505
